packages/xo-web/src/xo-app/hub/resource-form.js

```diff
@@ -52,9 +79,52 @@ export default decorate([
           onChange={effects.onChangePool}
           predicate={poolPredicate}
           required
-          value={state.pools}
+          value={pools}
         />
       </FormGrid.Row>
+      {install && isMultiplePools ? (
+        <div>
+          <SingleLineRow>
+            <Col size={6}>
+              <strong>{_('pool')}</strong>
+            </Col>
+            <Col size={6}>
+              <strong>{_('vdiSr')}</strong>
+            </Col>
+          </SingleLineRow>
+          <hr />
+          {pools.map(pool => (
+            <SingleLineRow key={pool.uuid} className='mt-1'>
+              <Col size={6}>{pool.name_label}</Col>
```

Comment:
You can use https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/render-xo-item.js#L46.